README.md

```diff
@@ -10,3 +10,14 @@ abandoned completely and merged into node_exporter**
 This currently exposes all basic stats (vdev_stats) and most extended stats (vdev_stats_ex) on a
 vdev-level. It doesn't expose per-disk or per-zpool stats, even though these are also available from
 the underlying API.
+
+
+To build with version info:
```

Comment:
Could you make this a markdown section?

Reply:
done

Reply:
Better, just name the section `Building with version information`

Reply:
done